joint_trajectory_controller/test/test_trajectory_controller.cpp

```diff
@@ -1233,8 +1238,39 @@ TEST_P(TrajectoryControllerTestParameterized, invalid_message)
   EXPECT_FALSE(traj_controller_->validate_trajectory_msg(traj_msg));
 }
 
-/// With allow_integration_in_goal_trajectories parameter trajectory missing position or
-/// velocities are accepted
+/**
+ * @brief invalid_message_nonzero_vel Test invalid velocity at trajectory end
+ */
+TEST_P(TrajectoryControllerTestParameterized, invalid_message_nonzero_vel)
```

Comment:
Just a bit more clarity.
```suggestion
TEST_P(
  TrajectoryControllerTestParameterized,
  expect_invalid_when_message_with_nonzero_end_velocity_and_when_param_false)
```